blueprints/app/files/app/app.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+import Resolver from 'ember-resolver';
+import loadInitializers from 'ember/load-initializers';
+import config from './config/environment';
+
+let App;
+
+Ember.MODEL_FACTORY_INJECTIONS = true;
```

Comment:
Sure, as I mentioned in https://github.com/emberjs/ember.js/pull/12412#issuecomment-144505248 we will need to make a blueprint for this (and likely `router.js`) then invoke that blueprint from `app`.  The `app.js` blueprint lives in Ember, but the `app` structure blueprint lives in ember-cli...